lib/ddtrace/configuration/components.rb

```diff
@@ -72,6 +74,17 @@ def build_tracer(settings)
           tracer
         end
 
+        def build_profiler(settings)
+          return unless Datadog::Profiling.supported?
+
+          recorder = build_profiler_recorder(settings)
+          collectors = build_profiler_collectors(settings, recorder)
+          exporters = build_profiler_exporters(settings)
+          scheduler = build_profiler_scheduler(settings, recorder, exporters)
+
+          Datadog::Profiler.new(collectors, scheduler)
```

Comment:
The profiler has to be (re)started here, if it was previously started.

Otherwise calling `Datadog.configure` will disable a previously active profiler.

Reply:
Right, discovered this limitation earlier, and this still needs to be addressed. Thanks for identifying it.

Reply:
This should be fixed now; when reconfigured, the profiler will automatically start if `enabled` is set.